frontend/packages/dev-console/src/components/import/__tests__/deployImage-submit-utils-data.ts

```diff
@@ -786,6 +825,12 @@ export const internalImageData: DeployImageFormData = {
       maxpods: '',
       concurrencytarget: '',
       concurrencylimit: '',
+      autoscale: {
+        autoscalewindow: '',
+        autoscalewindowUnit: '',
+        defaultAutoscalewindowUnit: '',
+      },
+      concurrencyutilization: '',
```

Comment:
it'll make sense to have a util to get mock initial values for serverless , this will avoid duplicate in other test files